src/js/editor/post.js

```diff
@@ -674,6 +674,25 @@ class PostEditor {
     return this.moveSectionBefore(collection, renderedSection, beforeSection);
   }
 
+  insertText(position, text) {
+    let {marker,offset} = position.markerPosition;
+    let section = position.section;
+    let nextPosition = position;
```

Comment:
this could early-return if the section is not markerable, which could improve the clarity:
```
if (!section.isMarkerable) {
  return position;
}
```